main.go

```diff
@@ -34,6 +37,13 @@ const APIOffsetAddr uintptr = 0x33EE420 // 1.09
 const GGStriveAPIURL = "https://ggst-game.guiltygear.com/api/"
 const PatchedAPIURL = "http://127.0.0.1:21611/api/"
 
+const GithubDownloadURL = "https://github.com/optix2000/totsugeki/releases/latest/download/"
```

Comment:
You can get the download URL from the API instead of hard coding, but it's not a big deal to leave it.